creator-node/src/components/replicaSet/syncQueueComponentService.js

```diff
@@ -1,30 +1,19 @@
 /**
  * Enqueues sync operation into syncQueue for provided walletPublicKeys against provided creatorNodeEndpoint
+ * @param {Object} params See the SyncQueue for explicit params
```

Comment:
hahaha ok this isn't a jsdoc

Reply:
ya but like why repeat the docs when this is just a wrapper function

Reply:
I actually think this file is redundant. if we're intending these functions to process logic, we should move these into their own utils file and import it anywhere across the code base. we do that for async processing queue, image transcoding queue, and transcoding queue

Reply:
I would argue to remove this cus now we've created extra layer of abstraction around maintaining this file + where these functions are declared

Reply:
idk about all that - this file seems consistent with the component service pattern used in other files
my point is just that this jsdoc as is serves little purpose
w/e, not worth the time here

Reply:
I think we need to consolidate what it means to be a component. I think if it's just another wrapper fn around the original fn, it's pointless. I think our component breakdown in the content node repo needs a redesign. thats a larger question for later tho

Reply:
yeah i agree, worth some further thought later